lib/katello/repository_types/python.rb

```diff
@@ -20,9 +20,9 @@
   publication_class PulpPythonClient::PythonPythonPublication
   publications_api_class PulpPythonClient::PublicationsPypiApi
 
-  generic_remote_option :includes, type: Array, description: "A list containing project specifiers for Python packages to include."
-  generic_remote_option :excludes, type: Array, description: "A list containing project specifiers for Python packages to exclude."
-  generic_remote_option :package_types, type: Array, description: "A list of package types to sync for Python content. Leave blank to get every package type."
+  generic_remote_option :includes, type: Array, description: "A list containing project specifiers for Python packages to include.", input_type: "textarea", delimiter: "\\n"
+  generic_remote_option :excludes, type: Array, description: "A list containing project specifiers for Python packages to exclude.", input_type: "textarea", delimiter: "\\n"
+  generic_remote_option :package_types, type: Array, description: "A list of package types to sync for Python content. Leave blank to get every package type.", input_type: "text", delimiter: ","
```

Comment:
We should have a list of the available types and a note about what the delimiter character is.